examples/hotrod/services/driver/redis.go

```diff
@@ -22,20 +22,21 @@ import (
 	"math/rand"
 	"sync"
 
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
 	"go.uber.org/zap"
 
 	"github.com/jaegertracing/jaeger/examples/hotrod/pkg/delay"
 	"github.com/jaegertracing/jaeger/examples/hotrod/pkg/log"
-	"github.com/jaegertracing/jaeger/examples/hotrod/pkg/tracing"
```

Comment:
Removing this is causing the errors below. I think you need to keep it.

Reply:
Yes, but `tracing.Init()` doesn't return opentelemetry instance as defined in the struct.

Reply:
In your PR you mentioned `tracing.InitOTEL()` to  adjust the function call.

Reply:
Ah yup, you're right; sorry, wasn't sure what you were trying to achieve.

Reply:
added that missing tracing.InitOTEL() func.